tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -222,5 +254,61 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 				return releaseCreated.IsReleased()
 			}, releaseCreationTimeout, defaultInterval).Should(BeTrue())
 		})
+
+		It("validates that imageIds was pushed to pyxis are identical to those in task create-pyxis-image.", func() {
+			Eventually(func() bool {
+				isImageExist := 0
+				for _, imageId := range imageIDs {
```

Comment:
How about changing the order of `Eventually` and `for`? So that we do:
```
for _, imageId := range imageIDs {
    Eventually()
}
```
That way we wait for one image at a time and we verify one image at a time - rather than having to repeatedly ask Pyxis for both.